custom/conf/app.example.ini

```diff
@@ -1494,6 +1494,9 @@ PATH =
 ;;
 ;; Timeout for Sendmail
 ;SENDMAIL_TIMEOUT = 5m
+;;
+;; convert \r\n to \n for Sendmail
+;SENDMAIL_CONVERT_CRLF = true
```

Comment:
Are there any requirement to convert \n to \r\n ?

Reply:
Currently, the `gomail` package always output CRLF for EOL. So there is no case for \n to \r\n.

The key problem is the unmaintained gomail package now. If we could maintain it by ourselves, we would have a better solution.

Reply:
So let's just fork and fix that upstream.

Reply:
That is not going to be a solution for 1.16.

Reply:
> That is not going to be a solution for 1.16.

I see your thought. Can you help to explain the reason behind it (why it is not a solution for 1.16)? Thanks ~

Reply:
https://github.com/go-gomail/gomail/issues/142

uff the only fork is https://github.com/go-mail/mail (https://github.com/go-gomail/gomail/issues/108)

but now also stale as a owner kicked out the other ... who want to do heavy refactoring ( looks like they had two different ideas)

so YES we would have to fork it :X